scripts/cybershake/slack_alert.yaml.example

```diff
@@ -0,0 +1,7 @@
+# cybershake_progress.py depends on this file to send the summary to SLACK channel
+users:
+  - <@U1LDB2BRT>
+  - <@U0C0T55F0>
+  - <@U3S75KRUH>
+  - <@U3DTBKABF>
+url : https://hooks.slack.com/services/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
```

Comment:
Should we publicly share this URL? Might be a good idea to rewrite history to remove it - unsure if spambots search git for URLs like this?

Reply:
Oops

Reply:
Let's get all other issues addressed. When you and James are both happy, I will open another PR with this URL removed, and delete this PR

Reply:
@jasonmotha  I have invalidated the URL, so that spambots won't have much fun with it. (We don't have to re-open this PR as I suggested yesterday). Also added some info regarding how such info can be obtained.